### PR TITLE
Added an admin page that handles all admin functions

### LINKED ISSUE #477: [Frontend] - Admin Panel Page
There are admin specific features that should only be available through the 'admin panel' page.

**Description** 
Create a new page with tabs. "Admin Panel Page" each tab would be an existing admin only funcitonality (similar to user profile page).
The tabs should be: categories management, dimension management, admin management (maybe other tabs too, that were not taken into account)

**Additional Notes**
Not only that the Admin page should only be avaliable for admin and super users only. Category, Dimension and Admin management tabs of the "admin panel page" should only be accessible for superuser only. if the admin or user go to those tabs via URL they should get a "You dont have access to this page" message in a snackbar.

**Acceptance Criteria**

### PR DESCRIPTION
**Pull request by laithAwad**
Added admin page and removed those tabs from the profile page, also added a check for a super admin to view certain tabs

### CODE REVIEW

**Review by Mehrdadmaskull — APPROVED** [review]
Code looks nice and clean, LGTM

File: Client/src/Components/Admin/AdminPageView.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                                <Alert onClose={() => setAlertOpen(false)} severity="error">
+                                    You do not have access to this page
+                                </Alert>
+                            </Snackbar>}
+                    </TabPanel>
+                </Container>
+            </div>
+        </>
+    )
+}
```
Comment (Vatika1) [inline Client/src/Components/Admin/AdminPageView.tsx:73]: nice

File: Client/src/Components/Admin/AdminPageView.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    return (
+        <>
+            <div>
+                <AppBar position="static">
+                    <Tabs value={tab} onChange={handleChange}>
+                        <Tab label="Categories Management" {...a11yProps(0)} />
+                        <Tab label="Dimension Management" {...a11yProps(1)} />
+                        <Tab label="Admin Management" {...a11yProps(2)} />
+                    </Tabs>
```
Comment (Vatika1) [inline Client/src/Components/Admin/AdminPageView.tsx:24]: nice

**Review by Vatika1 — APPROVED** [review]
looks good to me. good work

File: Client/src/Common/Consts/Routes.tsx [hunk 3]
```diff
@@ -44,5 +45,6 @@ export const routes: Routes = {
   signUpRoute: { route: '/sign-up', permission: 'anyone' },
   forgotPasswordRoute: { route: '/api/v1/forgotPassword', permission: 'anyone' },
   cellSizeAnalysisRoute: { route: '/cellSizeAnalysis', permission: 'anyone' },
-  resetPasswordRoute: { route: '/api/v1/resetPassword/:resetToken?', permission: 'anyone' }
+  resetPasswordRoute: { route: '/api/v1/resetPassword/:resetToken?', permission: 'anyone' },
+  adminPageRoute: { route: '/adminPage', permission: 1 }
```
Comment (ds161833) [inline Client/src/Common/Consts/Routes.tsx]: can you change the permissions to 2

**Review by ds161833 — CHANGES_REQUESTED** [review]

File: Client/src/Common/Consts/Routes.tsx (hunk 3 above)
Comment (laithAwad) [reply Client/src/Common/Consts/Routes.tsx]: done

**Review by ds161833 — APPROVED** [review]

**Pull request merged**